samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -173,13 +176,21 @@ Testing
 #. On the Android device, go to :guilabel:`Settings` > :guilabel:`Google` > :guilabel:`Devices & sharing` (or :guilabel:`Device connections`, depending on your Android device configuration) > :guilabel:`Devices`.
 #. Move the Android device close to the Fast Pair Provider that is advertising.
 #. Wait for Android device's notification about the detected Fast Pair Provider.
-   The notification is similar to the following one:
+   If you use Model ID certified by Google, the notification is similar to the following one:
 
    .. figure:: /images/bt_fast_pair_discoverable_notification.png
       :scale: 33 %
       :alt: Fast Pair discoverable advertising Android notification
 
    The device model name and displayed logo depend on the data provided during the device model registration.
+
+   If you use debug Model ID (e.g. default "NCS Fast Pair demo"), the notification is similar to the following one:
+
+   .. figure:: /images/bt_fast_pair_discoverable_notification_debug.png
```

Comment:
@divipillai Should regular images have any specific frames?

Reply:
yeah .png needs to be Width: 780 px and Height: 800 px as per the guidelines.

Reply:
Those are screenshots so the proportions (height to width ratio) depend really on what I want to show in the picture. Widening or lengthening the picture would look unnatural. Am I missing something?

Reply:
In this case I think it's fine if we just make sure that the image parameters don't exceed the sizes mentioned by Divya. Would be great to get them closer to these parameters but without affecting the original shape of the image